packages/core/demo/demo-data/line.ts

```diff
@@ -55,7 +55,8 @@ export const curvedLineOptions = {
 			}
 		}
 	},
-	curve: "curveNatural",
+	curve: "curveCardinal",
+	curveOptions: { tension: 0 },
```

Comment:
I think expanding the `curve` field to:
```js
{
  type: "curveNatural",
  tension: 0,
  ...
}
```

Makes more sense here

Reply:
I added the new option to avoid a breaking change. Did you want to support both forms?

```
curve: "curveCardinal"
```
```
curve: {
  type: "curveCardinal",
  tension: 0
}
```

Most of the curves do not have any additional options available to them. (ie curveNatural has no tension, alpha, or anything.)

Reply:
Sure, let's support both, that's a good idea considering not all curves support additional options